app/assets/javascripts/discourse/app/components/composer-container.hbs

```diff
@@ -102,36 +102,7 @@
           />
         </div>
 
-        <ComposerEditor
-          @topic={{this.composer.topic}}
-          @composer={{this.composer.model}}
```

Comment:
`ComposerEditor`'s `this.composer` used to be the model, changed now to use the service. Occurrences that still need the model will reference it (`composer.model`) explicitly.